vcl_templates/www.vcl.erb

```diff
@@ -374,6 +374,34 @@ sub vcl_recv {
     return(pass);
   }
 
+  if (req.http.Cookie ~ "cookies_policy") {
```

Comment:
might be better to put this `if` block one step above and group together all the cookies check?

Reply:
Good shout :+1: